zingolib/src/lightclient.rs

```diff
@@ -260,6 +260,45 @@ impl LightClient {
     }
 }
 impl LightClient {
+    /// The wallet this fn associates with the lightclient is specifically derived from
+    /// a spend authority.
+    pub async fn new_from_wallet_base_async(
```

Comment:
Given that we're adding an `async_client` feature, why is this not `#[cfg(async_client)]`?